src/icf/README.md

```diff
@@ -0,0 +1,229 @@
+## Interface Call Forwarder
```

Comment:
can you add this to the documentation website as well

Reply:
will add it